nvflare/private/fed/utils/fed_utils.py

```diff
@@ -501,3 +501,19 @@ def get_scope_prop(scope_name: str, key: str) -> Any:
     check_str("key", key)
     data_bus = DataBus()
     return data_bus.get_data(_scope_prop_key(scope_name, key))
+
+
+def get_job_launcher(job_meta: dict, fl_ctx: FLContext) -> dict:
+    engine = fl_ctx.get_engine()
+
+    job_launcher_ctx = engine.new_context()
+    # Remove the potential not cleaned up JOB_LAUNCHER
+    job_launcher_ctx.remove_prop(FLContextKey.JOB_LAUNCHER)
```

Comment:
This is unnecessary since job_launcher_ctx is a new context.

Reply:
I added this to handle the case if some components accidentally set the JOB_LAUNCHER as a sticky in the FLConext.